File: INSERCIONES INICIO,ESP,FINAL/insercion.cs
using System;
using System.Collections.Generic;

class Program{
    static void Main(){
    int ele=52;

    //insercion al principio
    List<int> arr1= new List<int>{11,21,31,41,51,61};
    Console.WriteLine("antes: "+string.Join(",",arr1));
    arr1.Insert(0,ele);
    Console.WriteLine("despues: "+string.Join(",",arr1));

    //insercion especifico
    List<int> arr2= new List<int>{11,21,31,41,51,61};
    arr2.Insert(3,ele);
    Console.WriteLine("ahora: "+string.Join(",",arr2));

    //insercion al final
    List<int> arr3= new List<int>{11,21,31,41,51,61};
    int eleFinal=72;
    arr3.Add(eleFinal);
    Console.WriteLine("final: "+string.Join(",",arr3));
    }
}

File: INSERCIONES INICIO,ESP,FINAL/insercion.java
import java.util.ArrayList;
import java.util.Arrays;

public class insercion {
    public static void main(String[] args) {
        int ele=52;

        //insercion al principio
        ArrayList<Integer> arr1 = new ArrayList<>(Arrays.asList(11,21,31,41,51));
        System.out.println("antes: " + arr1);
        arr1.add(0, ele);
        System.out.println("despues: " + arr1);

        //insercion especfica
        ArrayList<Integer> arr2 = new ArrayList<>(Arrays.asList(11,21,31,41,51));
        arr2.add(3, ele);
        System.out.println("despues: " + arr2);

        //insercion al final
        ArrayList<Integer> arr3 = new ArrayList<>(Arrays.asList(11,21,31,41,51));
        ele=72;
        arr3.add(ele);
        System.out.println("despues: " + arr3);
    }   
}